apps/easypid/src/features/wallet/FunkeOfflineQrScreen.tsx

```diff
@@ -126,12 +127,12 @@ export function FunkeOfflineQrScreen() {
     })
   )
 
-  useEffect(() => {
-    // Cleanup function that runs when component unmounts
-    return () => {
-      shutdownDataTransfer()
-    }
-  }, [])
+  // useEffect(() => {
```

Comment:
Broken?

Reply:
No this is bugged, it shutsdown after we navigate, but we must keep it alive and shut it down later.